src/main/java/io/github/opencubicchunks/worldpainterplugin/CubicLevel.java

```diff
@@ -148,15 +148,15 @@ public int getGameType() {
     }
 
     public boolean isHardcore() {
-        return getBoolean(TAG_HARDCORE);
+        return getBoolean(TAG_HARDCORE_);
     }
 
     public String getGeneratorName() {
-        return getString(TAG_GENERATOR_NAME);
+        return getString(TAG_GENERATOR_NAME_);
     }
 
     public int getGeneratorVersion() {
-        return getInt(TAG_GENERATOR_VERSION);
+        return getInt(TAG_GENERATOR_VERSION_);
```

Comment:
What happened there?

Reply:
Some Minecraft ids have versions that start with a capital letter _and_ that start with a lowercase letter. I normalised the names of the constants so that a trailing underscore means it's the lowercase version.

Reply:
Or rather that it is lowercase, regardless of whether there is another version...